android-sdk/src/main/java/com/checkout/android_sdk/Utils/CheckoutThemeInterface.kt

```diff
@@ -0,0 +1,10 @@
+package com.checkout.android_sdk.Utils
+
+internal interface CheckoutThemeInterface {
+    fun getColorPrimaryProperty(): MutableMap<String, Any?>
+    fun getColorAccentProperty(): MutableMap<String, Any?>
+    fun getColorButtonNormalProperty(): MutableMap<String, Any?>
+    fun getTextColorPrimaryProperty(): MutableMap<String, Any?>
+    fun getColorControlActivatedProperty(): MutableMap<String, Any?>
+    fun getColorControlNormalProperty(): MutableMap<String, Any?>
```

Comment:
From the first look these functions look more like getters.
If it's so, is there any need for map mutability in this case?

Reply:
It's not required while obtaining data. Using the same data type while adding it. However, I will use Map for it.